llvm/tools/llvm-profgen/ProfileGenerator.h

```diff
@@ -121,7 +121,7 @@ class ProfileGeneratorBase {
   double calculateDensity(const SampleProfileMap &Profiles,
```

Comment:
If we change how density is calculated, we should change it everywhere to be consistent. Why leaving the old heuristic there for non-CS profile?

Reply:
Yeah, I planned a new another diff, new it's updated in the new version.